doc/check-options.md

```diff
@@ -0,0 +1,215 @@
+| `selector`          | `footer:not([role]), [role=contentinfo]` | Selector used to determine if a page has a contentinfo landmark                            |
+| `nativeScopeFilter` | `article, aside, main, nav, section`     | Selector used to ignore contentinfo landmarks that have a parent that matches the selector |
+
+### page-no-duplicate-main
+
+| Option     | Default                           | Description                                              |
+| ---------- | :-------------------------------- | :------------------------------------------------------- |
+| `selector` | `main:not([role]), [role='main']` | Selector used to determine if a page has a main landmark |
+
+### duplicate-img-label
+
+| Option           | Default                                         | Description                                                                           |
+| ---------------- | :---------------------------------------------- | :------------------------------------------------------------------------------------ |
+| `parentSelector` | `button, [role=button], a[href], p, li, td, th` | Selector used to look for the parent of an image that could duplicate the image label |
+
+### label-content-name-mismatch
+
+| Option               | Default | Description                                                                                                                                                               |
+| -------------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `pixelThreshold`     | `0.1`   | Percent of differences in pixel data or pixel width needed to determine if a font is a ligature font (ligature fonts are ignored when comparing the label to the content) |
+| `occuranceThreshold` | `3`     | Number of times the font is encountered before auto-assigning the font as a ligature or not                                                                               |
+
+### has-lang
+
+| Option       | Default                | Description                         |
+| ------------ | :--------------------- | :---------------------------------- |
+| `attributes` | `['lang', 'xml:lang']` | Attributes to check for lang values |
+
+### valid-lang
+
+| Option       | Default                | Description                               |
+| ------------ | :--------------------- | :---------------------------------------- |
+| `attributes` | `['lang', 'xml:lang']` | Attributes to check for valid lang values |
+
+### frame-tested
+
+| Option        | Default | Description                                                |
+| ------------- | :------ | :--------------------------------------------------------- |
+| `isViolation` | `false` | If an iframe without axe should be reported as a violation |
```

Comment:
```suggestion
| `isViolation` | `false` | If an `iframe` without axe should be reported as a violation |
```
What does "without axe" mean? an `iframe` that has not been tested by axe? An `iframe` that has not been tested by axe yet, but will be somewhere else?